cmobase/background.adoc

```diff
@@ -407,6 +407,7 @@ periodic cancellation of a reservation on another hart._
 * cache block size for management and prefetch
 * cache block size for zero
 * CBIE support at each privilege level
+* what happens if CBO operations are executed on addresses that are not cached?
```

Comment:
I think this is primarily relevant in the context of changing PMAs, and we definitely need sequences/litmus tests/etc. to describe how CMOs can be used to facilitate such changes. For static and consistent PMAs, I'm not sure additional specification is necessary, but I could be convinced otherwise....

FWIW, I will create an Open Issues section and move this there.